### DISCUSSION TITLE
Can the community Dicussions for GitHub Education be divided from classroom?

### DISCUSSION THREAD

**Discussion by famosab** [community/community#129984] [GitHub Education]
### Select Topic Area

Question

### Body

The community Diskussion forum is quite useful. As a teacher, I wish that the whole functionality of GitHub classroom was divided from the general discussion about GitHub Education. Given the fact that there is no repository to be found for the whole classroom implementation its really hard to get more information on other people having similar problems and also quickly realizing that there are new topics coming up regarding classroom. I know that there is a label but for me the whole classroom functionality is sort of a different topic than GitHub Education (which is often about benefits and so on).

**Comment by samus-aran** [comment]
Hi @famosab 👋 

We do have a private repo for verified Educators that has a category specifically to GitHub classroom. I can see that you are verified as a student and probably why you are unable to see it. If you wish to verify as a teacher let me know and you will be asked to re-verify via the application page.

In regards to making Classroom conversations more visible, yes we do heavily use labels. "Classroom" is a combination of bugs, questions, etc. You make a valid point that perhaps these discussions around Classroom can be better categorised. I'll take this back to the internal team and see how we can make it easier for the community to identify these conversational topics versus feature updates and bug reporting. 

Your feedback is extremely appreciated 👍

**Reply by famosab** [reply]
Sounds good! 
Regarding the verification: I was wondering how to verifiy as I am both a (PhD) student and teach classes. Most of the time I act more as a student than a teacher but for the classes where I do teach having access to the described repo would be fantastic.

**Reply by john-french** [reply]
Hi @samus-aran. Where is this private repo for verified Educators you speak of? I too find it hard to locate Github Classroom discussions since the move to the Github Education community repo

**Reply by john-french** [reply]
Classroom doesn't show up in the list of Product Categories either which makes these discussions hard to find
![image](https://github.com/user-attachments/assets/362205b1-9763-4b1c-ab63-d3e07cac74e3)

**Comment by john-french** [comment]
Ah, I see I need to be a member of Global Campus Teachers. I got an invite but didn't accept it in time. Is there any chance I could get sent another invite please @samus-aran ? (and I promise to accept it in time this time!)
